ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2687,6 +2687,9 @@ router_bgp:
       rd: < route distinguisher >
       route_targets:
         import_export: < route target >
+      mpls_control_word: < true | false, Default -> false >
+      label_flow: < true | false, Default -> false >
+      mtu: < 1-65535, Default -> ignore mtu >
```

Comment:
IMO, its not required to say that mtu will be ignored if not defined. Also, I am not completely sure if we are defining default behavior in eos_cli_config_gen role. Usually, we skip that.

Reply:
True, fixed!